src/Tribe/I18n.php

```diff
@@ -190,13 +250,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ( $flags & static::COMPILE_INPUT ) {
+						$strings[ $key ][] = __( $value );
+					}
+					if ( $flags & static::COMPILE_STRTOLOWER ) {
+						$strings[ $key ][] = __( strtolower( $value ) );
+					}
+					if ( $flags & static::COMPILE_UCFIRST ) {
+						$strings[ $key ][] = __( ucfirst( $value ) );
+					}
 				} else {
-					$strings[ $key ][] = __( $value, $domain );
-					$strings[ $key ][] = __( strtolower( $value ), $domain );
-					$strings[ $key ][] = __( ucfirst( $value ), $domain );
+					if ( $flags & static::COMPILE_INPUT ) {
+						$strings[ $key ][] = __( $value, $domain );
+					}
+					if ( $flags & static::COMPILE_STRTOLOWER ) {
+						$strings[ $key ][] = __( strtolower( $value ), $domain );
```

Comment:
* :no_entry_sign: The $text arg must be a single string literal, not "strtolower( $value )".
* :no_entry_sign: The $domain arg must be a single string literal, not "$domain".

<sub>hash: dbe4dcaaac22b1b91eb81b5450f86097</sub>